tiller/templates/test/test-pod.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
 spec:
   containers:
     - name: tiller-deploy-test
-      image: chatwork/helm:2.10.0
+      image: chatwork/helm:2.12.2
```

Comment:
@cw-sakamoto 

```
 helm init --dry-run -o yaml
```

Please check that the resources are the same for `2.10.0` and `2.12.2`.